packages/manager/modules/netapp/src/dashboard/volumes/dashboard/snapshots/delete/template.html

```diff
@@ -0,0 +1,34 @@
+<form
+    id="deleteSnapshot"
+    name="deleteSnapshot"
+    data-ng-submit="deleteSnapshot.$valid && $ctrl.deleteSnapshot()"
```

Comment:
No field in this form, so no way to get invalid here right ?
```suggestion
    data-ng-submit="$ctrl.deleteSnapshot()"
```